stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api2/MoshiChatApi.kt

```diff
@@ -1,6 +1,13 @@
 package io.getstream.chat.android.client.api2
 
 import io.getstream.chat.android.client.api.ChatApi
+import io.getstream.chat.android.client.api.ErrorCall
+import io.getstream.chat.android.client.api.models.FilterObject
+import io.getstream.chat.android.client.api.models.QueryChannelRequest
+import io.getstream.chat.android.client.api.models.QueryChannelsRequest
+import io.getstream.chat.android.client.api.models.QuerySort
+import io.getstream.chat.android.client.api.models.QueryUsersRequest
+import io.getstream.chat.android.client.api.models.SearchMessagesRequest
```

Comment:
Really unfortunate, but these old network models are part of the API of the ChatAPI interface, so we have to reference them in the Moshi impl too, before converting them to new, proper DTOs